src/plays/color-palettes-generator/components/SingleColor.js

```diff
@@ -0,0 +1,23 @@
+import { alertToast } from './utils';
+
+const SingleColor = ({ index, hexColor }) => {
+  return (
+    <>
+      <article
+        className={`color-palettes-generator-color ${
+          index > 10 && 'color-palettes-generator-color-light'
+        }`}
+        style={{ backgroundColor: `${hexColor}` }}
+        onClick={() => {
+          navigator.clipboard.writeText(hexColor);
+          alertToast(`${hexColor} Copied!`);
+        }}
```

Comment:
This is an async operation so make sure to wrap it in try-catch.

You can create a handler and move this logic there.

```suggestion
        onClick={handleCopyHexCode}
```

Reply:
Done